packages/iTwinUI-react/src/core/ComboBox/ComboBoxMenu.tsx

```diff
@@ -56,31 +56,27 @@ const VirtualizedComboBoxMenu = React.forwardRef(
       scrollToIndex: focusedVisibleIndex,
     });
 
-    const overflowY = getWindow()?.CSS?.supports?.('overflow-x: overlay')
-      ? { overflowY: 'overlay' }
-      : { overflowY: 'auto' };
+    const isOverflowOverlaySupported =
```

Comment:
to same comment, this could be moved outside the component code too

Reply:
done